assembly/guide-configs/strongly-typed-errors.json

```diff
@@ -0,0 +1,152 @@
+{
+  "file_name": "strongly-typed-errors.md",
+  "id": "strongly-typed-errors",
+  "file_dir": "/",
+  "title": "Strongly Typed Errors",
+  "sidebar_label": "Strongly Typed Errors",
```

Comment:
Consider "Errors" here as well - 

"Strongly typed" is more of an internal project name.